src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1386,6 +1392,25 @@ void NotationActionController::toggleScoreConfig(ScoreConfigType configType)
     interaction->setScoreConfig(config);
 }
 
+void NotationActionController::playSelectedElement()
+{
+    auto interaction = currentNotationInteraction();
+    if (!interaction) {
+        return;
+    }
+
+    Element* element = interaction->selection()->element();
+    if (!element || !element->isNote()) {
+        return;
+    }
+
+    if (playbackConfiguration()->playChordWhenEditing()) {
+        element = element->elementBase();
+    }
+
+    playbackController()->playElementOnClick(element);
```

Comment:
I think now it is better to rename the method `playElementOnClick` to like `playElement` :)

Reply:
Renamed